libs/application/templates/financial-aid/src/lib/messages/inRelationship.ts

```diff
@@ -0,0 +1,51 @@
+import { defineMessages } from 'react-intl'
+
+// Reason
+export const inRelationship = {
+  general: defineMessages({
+    sectionTitle: {
+      id: 'fa.application:section.personalInterest.inRelationship.sectionTitle',
+      defaultMessage: 'Hjúskaparstaða',
+      description: 'In relationship section title',
+    },
+    pageTitle: {
+      id: 'fa.application:section.personalInterest.inRelationship.pageTitle',
+      defaultMessage: 'Hjúskaparstaða þín',
+      description: 'In relationship page title',
+    },
+    intro: {
+      id: 'fa.application:section.personalInterest.inRelationship.intro',
+      defaultMessage:
+        'Þar sem þú ert í sambúð þarft þú að skila inn umsókn um fjárhagsaðstoð og maki þinn að skila inn upplýsingum um tekjur.',
+      description: 'In relationship intro',
+    },
+    description: {
+      id:
+        'fa.application:section.personalInterest.inRelationship.description#markdown',
+      defaultMessage:
+        'Hvað þýðir það? Þú klárar að fylla út þína umsókn um fjárhagsaðstoð hér og maki þinn notar sín rafrænu skilríki til að skila inn nauðsynlegum gögnum. \\n\\n\\n Úrvinnsla umsóknarinnar hefst þegar öll gögn hafa borist.',
```

Comment:
this \\n\\n\\n is not working, I saw it on the screenshot, it should be \n\n\n I think 🤔

Reply:
You are writing the same thing twice 😄

Reply:
But fixed :)